lib/SIL/Parser/ParseSIL.cpp

```diff
@@ -1244,7 +1238,7 @@ bool SILParser::parseSILType(SILType &Result,
   
   // Apply attributes to the type.
   auto *attrRepr = P.applyAttributeToType(TyR.get(), attrs, specifier, specifierLoc);
-  auto Ty = performTypeLocChecking(attrRepr, /*IsSILType=*/true, OuterGenericEnv);
```

Comment:
This needs to use the full call in order to propagate `TypeResolutionFlags::SILType`.

Reply:
Oops